clients/fides-js/src/fides.ts

```diff
@@ -124,12 +168,21 @@ async function init(this: FidesGlobal, providedConfig?: FidesConfig) {
     consentPrefsOverrides,
     experienceTranslationOverrides,
   };
+
+  // Check for an existing cookie for this device
+  const existingFidesCookie: FidesCookie | undefined = getFidesConsentCookie();
+  let consentFromOneTrust: NoticeConsent | undefined;
+  if (!existingFidesCookie && optionsOverrides.otFidesMapping) {
+    consentFromOneTrust = readConsentFromOneTrust(config, optionsOverrides);
```

Comment:
we only allow overriding fides with OT if 
1. we don't already have a fides cookie and 
2. we have a mapping obj defined